src/style-spec/reference/v8.json

```diff
@@ -3673,6 +3677,112 @@
       }
     }
   },
+  "fog": {
+    "range": {
+      "type": "array",
+      "default": [
+        2,
+        12
+      ],
+      "minimum": 0,
+      "maximum": 20,
+      "length": 2,
+      "value": "number",
+      "property-type": "data-constant",
+      "transition": true,
+      "expression": {
+        "interpolated": true,
+        "parameters": [
+          "zoom"
+        ]
+      },
+      "doc": "The near and far limits of the fog layer. Units are relative to the map height, so that 2.0 corresponds to twice the map height, in pixels.",
```

Comment:
Can this new unit type be named and defined so that it can be referenced uniformly across other style properties or in API docs as well

Reply:
If we multiply by 100, it's [CSS `vh` units](https://developer.mozilla.org/en-US/docs/Web/CSS/length#vh), but otherwise I'm not sure the best name for it.

Reply:
> The near and far limits of the fog layer.

This suggests that fog only exists within the range. It might be good to document that this is the range that fog fades in over